src/presentation/test/integration/registration/address/confirm-registered-office-address.test.ts

```diff
@@ -38,15 +38,15 @@ describe("Confirm Registered Office Address Page", () => {
   });
 
   describe("GET Confirm Registered Office Address Page", () => {
-    it("should load the confirm registered office address page with English text", async () => {
+    it.only("should load the confirm registered office address page with English text", async () => {
```

Comment:
test has a .only - i guess this can be removed

Reply:
Removed